src/spyglass/utils/dj_merge_tables.py

```diff
@@ -530,7 +521,7 @@ def fetch_nwb(
         """
         if isinstance(self, dict):
             raise ValueError("Try replacing Merge.method with Merge().method")
-        if restriction == True and self.restriction:
+        if restriction is True and self.restriction:
```

Comment:
What's the logic behind using `is` vs. `==`? Is there an issue for testing for the exact same object vs. equality?

Reply:
It was flagged by my linter as PEP8 E712